scripts/export-meshseg-onnx.py

```
@@ -16,161 +16,467 @@
+    for i in range(samples):
+        P, L = sample_body(rng)
+        if rng.random() < 0.5: P, L = mirror(P, L)
+        aP[i] = P; aL[i] = L
+    return aP, aL
+
+
+# --- mined real meshes (rig-prior ground truth) ------------------------------
+def canonicalise(P, L, fname=""):
+    """Rotate a mined cloud into the canonical frame using its own labels:
+    up = the legs' own principal axis (robust to tails skewing the torso
+    centroid), left = right→left limb axis, forward = left×up, with a 180°
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)